tICA/README.md

```diff
@@ -0,0 +1,19 @@
+# Temporal ICA
+
+This pipeline uses ICA on a large amount of fMRI data (typically a large group of subjects),
+in order to separate global nuisance effects (such as due to respiration) from neural signals that
+have a nonzero spatial mean.
+
+The processing modes also allow for applying temporal ICA cleanup to datasets that are not
+large enough to reliably produce a new temporal ICA decomposition.  Briefly, the processing
+modes are:
+
+- NEW - do a full pipeline run from scratch, starting with a fresh MIGP, intended for large datasets
```

Comment:
```suggestion
- NEW - do a full pipeline run from scratch, starting with a fresh group-PCA (using "MIGP"), intended for large datasets
```

Reply:
I switched these all to "group-PCA" with a MIGP reference in only the first.